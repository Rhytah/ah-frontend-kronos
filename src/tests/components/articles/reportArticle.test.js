/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import { shallow } from 'enzyme';
import ReportArticle from '../../../components/articles/report/reportArticle';


describe('<ReportArticle />', () => {
  it('should match snapshot when loading', () => {
    const props = {
      showLoader: true,
      handleFormSubmission: jest.fn(),
      closeTheReportForm: jest.fn(),
      onInputChange: jest.fn(),

    };
    const wrapper = shallow(<ReportArticle {...props} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should match snapshot when not loading', () => {
    const props = {
      showLoader: false,
      handleFormSubmission: jest.fn(),
      closeTheReportForm: jest.fn(),
      onInputChange: jest.fn(),

    };
    const wrapper = shallow(<ReportArticle {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
