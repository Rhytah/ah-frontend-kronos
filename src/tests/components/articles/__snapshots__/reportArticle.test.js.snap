// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReportArticle /> should match snapshot when loading 1`] = `ShallowWrapper {}`;

exports[`<ReportArticle /> should match snapshot when not loading 1`] = `ShallowWrapper {}`;
